### ISSUE TITLE
[Examples] Solve the problem of cross origin requests in integrating with public data sources

### ISSUE THREAD

**Issue by akhenry** [nasa/openmct#417]
In the rems_data branch, integration with the public web service requires cross origin requests. These are rejected by default because the public server does not return an Access-Control-Allow-Origin CORS response header. Because we cannot rely on public data sources to always provide an appropriate Access-Control-Allow-Origin header, we need a solution that allows examples that integrate with public data sources to be run regardless.

Suggest that we add a proxying service to app.js that would proxy requests to remote resources. Requests to remote resources from the examples would instead go to localhost and be forwarded to the remote data source. The alternative is to run browser with cross origin security disabled, however I would prefer a solution that works out of the box so that people downloading openmctweb can get up and running quickly without having to modify browser settings.

**Comment by larkin** [comment]
We have this support in an alternative branch, could migrate it here.

**Comment by akhenry** [comment]
Which branch? Is it in open?

**Comment by larkin** [comment]
@akhenry Not open currently but it's real simple, `npm install --save request`, add a `var request = require('request');`, and then add this route:

```javascript
    app.use('/proxyUrl', function proxyRequest(req, res, next) {
        console.log('Proxying request to: ', req.query.url);
        req.pipe(request({
            url: req.query.url,
            strictSSL: false
        }).on('error', next)).pipe(res);
    });
```

**Comment by akhenry** [comment]
Implemented by #637
